libraries/AP_HAL/board/linux.h

```diff
@@ -227,6 +227,7 @@
 #define HAL_BARO_DEFAULT HAL_BARO_MS5611_SPI
 #define HAL_BARO_MS5611_NAME "ms5611"
 #define HAL_COMPASS_DEFAULT HAL_COMPASS_BBBMINI
+#define HAL_AIRSPEED_MS4515DO_I2C_BUS 2
```

Comment:
Hi @OXINARF ,

and there is a detail about it, for instance, on BB Black the i2c bus is 0 and 2, there is no i2c-1, and I don't know if other boards has this particularity.